src/nccl_ofi_rdma.c

```diff
@@ -4643,6 +4643,7 @@ static int send(nccl_net_ofi_send_comm_t *send_comm, void *data, int size, int t
 	 * props->maxRecvs > 1.
 	 */
 
+	bool polled_cq = false;
```

Comment:
why here in the middle of the function? would prefer to see it either put it at the top or put remotely near to where it's used

Reply:
this breaks under C++ because gotos cannot jump over initializers.